com.unity.render-pipelines.core/Tests/Editor/Volumes/VolumeComponentTests.cs

```diff
@@ -180,5 +180,66 @@ public void TestSupportedOnAvoidedIfHideInInspector()
             Assert.NotNull(volumeComponents);
             Assert.False(volumeComponents.Any());
         }
+
+        static private void TestAnimationCurveInterp(AnimationCurve lhsCurve, AnimationCurve rhsCurve, float t, float startTime, float endTime, int numSteps, float eps, bool debugPrint)
+        {
+            AnimationCurve midCurve = new AnimationCurve(lhsCurve.keys);
+            KeyframeUtility.InterpAnimationCurve(ref midCurve, rhsCurve, t);
+
+            for (int i = 0; i <= numSteps; i++)
+            {
+                float timeT = ((float)i) / ((float)numSteps);
```

Comment:
Do you need both casts?

Reply:
If we only do one of the casts, both casts will stall have to happen since the compiler will silently convert one of the ints to a float. So I wrote it this way to make it more explicit. But I don't have strong feelings either way.